### PR TITLE
BLOG(SENDING PR AGAIN):HOW THE INTERNET WORKS

### PR DESCRIPTION
**Pull request by adityachirania**
ALL CHANGES DONE
Screenshot (16)
Screenshot (17)
Screenshot (23)
Screenshot (20)
Screenshot (21)
Screenshot (22)

### CODE REVIEW

File: blog_src/_data/authors.yml [hunk 1]
```diff
@@ -757,4 +757,9 @@ nimbus98:
   github: nimbus98
   author_image: 'akashnair.jpeg'
   user_description: 'I like Food'
-  email: akash30121998@gmail.com
\ No newline at end of file
+  email: akash30121998@gmail.com
+adityachirania:
+  name: Aditya Chirania
+  github: adityachirania
+  author_image: blog-author.jpg
```
Comment (salman-bhai) [inline blog_src/_data/authors.yml]: Umm you can add your image in place of this 😛

File: blog_src/_posts/2019-10-02-How the internet works.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
...1970, ARPANET had grown to 13 different locations including HARVARD and MIT. By 1982, the network only had about 100 nodes. The increasing rush on the ARPANET led to the creation of TCP/IP. During the 1980s, the National Science Network funded several supercomputing centres around the United States. And in 1986 the agency created a TCP/IP-based network called NSFNET to link those supercomputing centres together and allow researchers across the country to use them. The primary goal was to allow computer science researchers to log into the supercomputers and perform academic research. But NSF decided not to limit NSFNET to that purpose, allowing the network to be used for a wide variety of academic purposes. As a result, the NSFNET became the internet's "backbone," the high-speed, long-distance network that allowed different parts of the internet to communicate. Soon, commercial firms took over the job of carrying long-distance internet traffic, allowing the government-funded NSFNET to be decommissioned. Officials were careful to ensure that no single company controlled too much of the backbone, helping to create a competitive market for internet connectivity that still exists today.
```
Comment (salman-bhai) [inline blog_src/_posts/2019-10-02-How the internet works.md]: Full form of ARPANET and link to ARPANET would be helpful here. In addition to that can you split it into two paragraphs?

File: blog_src/_posts/2019-10-02-How the internet works.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+comments: true
+---
+
+## EVER WONDERED HOW THE INTERNET WORKS?
+
+From the instant I search for a website to the instant the webpage loads on our browser, there are a lot of things that happens in between these two instants. Let’s take the example of [Google](http://google.com/) which most of us have searched for. The browser does not know what [Google](http://google.com/) is, so it sends a message to the internet service provider asking if it knows what Google is. The internet service provider passes on the URL to the DNS servers. A DNS server is a computer server that contains a database of public IP addresses and their associated hostnames. The DNS server returns the IP address of the Google servers and now the browser makes a request to the Google servers for the webpage. So, the google servers send the HTML, CSS and JavaScript files back to the browser and the browser loads these files and then finally the webpage appears in front of us. So, if you directly search for the IP address of [Google](http://google.com/) your webpage will still load and that too a little faster because, now we would not need the DNS server to look up the website’s server address for us.
```
Comment (salman-bhai) [inline blog_src/_posts/2019-10-02-How the internet works.md]: You could split the part from `A DNS Server...` to the next paragraph

Comment (adityachirania) [reply blog_src/_posts/2019-10-02-How the internet works.md]: Ive made the changes

File: blog_src/_posts/2019-10-02-How the internet works.md (hunk 2 above)
Comment (adityachirania) [reply blog_src/_posts/2019-10-02-How the internet works.md]: i ve made the changes

**Comment by salman-bhai** [comment]
Resolve conflicts please

**Pull request closed without merging**